lib/core-integration/src/Test/Integration/Scenario/API/Byron/TransactionsNew.hs

```diff
@@ -0,0 +1,452 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
```

Comment:
This file should be removed, shouldn't it? It's been dropped in previous PR concluding construct tx...